pyproject.toml

```diff
@@ -1,5 +1,83 @@
-[tool.black]
-line-length = 99
+[build-system]
+requires = ["setuptools>=68.2.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "xocto"
+version = "4.9.0"
+requires-python = ">=3.9"
+description = "Kraken Technologies Python service utilities"
+readme = "README.md"
+authors = [
+  {name = "Kraken Technologies", email = "talent@octopus.energy"},
+]
+maintainers = [
+  {name = "Kraken Technologies", email = "talent@octopus.energy"},
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Topic :: Software Development :: Build Tools",
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
```

Comment:
We should add support for Python 3.12 at some point too.

Reply:
Agreed, created #121.